### ISSUE TITLE
Swift 4.2 conversion issue

### ISSUE THREAD

**Issue by RamwiseMatt** [pmick/RxAVFoundation#23]
Swift 4.2 has removed `kCMTimeZero` in favor of `CMTime.zero`. This is an issue for this library, because of this code (https://github.com/pmick/RxAVFoundation/blob/master/Source/CMTime+Helpers.swift):

```
extension CMTime {
    public static var zero: CMTime {
        return kCMTimeZero
    }
}
```

This results in a `Ambiguous use of 'zero'` compiler error in Xcode.

Is there a plan to work on swift 4.2 support? For this issue at least, the solution would be to simply remove the mentioned file from the codebase.

Thanks!

**Comment by OhKanghoon** [comment]
#24

**Comment by pmick** [comment]
I will get CI working, this PR merged, and a release out tonight. Thanks ✌️ 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by pmick** [comment]
Hey @RamwiseMatt @OhKanghoon I just published 2.1.0 which includes all of the swift 4.2 api changes to AVFoundation.

**Comment by OhKanghoon** [comment]
Great!
